pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -1042,23 +1025,11 @@ func isNUMAAffinitiesEqual(numaAffinity1, numaAffinity2 []int) bool {
 	return bitMask1.IsEqual(bitMask2)
 }
 
-func isAffinityViolatingNUMAAllocations(machineState state.NUMANodeMap, mask bitmask.BitMask) bool {
```

Comment:
I renamed parameter `mask` to `affinity` to make it consistent with the function name (is**Affinity**...)